source/expressions.tex

```diff
@@ -600,13 +600,13 @@
 
 \pnum
 When an lvalue-to-rvalue conversion
-is applied to an expression \tcode{e}, and either
+is applied to an expression $E$, and either
 \begin{itemize}
-\item \tcode{e} is not potentially evaluated, or
-\item the evaluation of \tcode{e} results in the evaluation of a member
-      \tcode{ex} of the set of potential results of \tcode{e}, and \tcode{ex}
+\item $E$ is not potentially evaluated, or
+\item the evaluation of $E$ results in the evaluation of a member
+      $E_x$ of the set of potential results of $E$, and $E_x$
       names a variable \tcode{x} that is not odr-used by
-      \tcode{ex}\iref{basic.def.odr},
+      $E_x$\iref{basic.def.odr},
```

Comment:
The subscript x should probably be in code font since that's the same `x` as the variable name.

Reply:
Fixed.